crypto/tls/tls_test.go

```diff
@@ -182,3 +183,22 @@ func TestGetTLSConfig_ServerName(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "myserver.com", tlsConfig.ServerName)
 }
+
+func TestGetTLSConfig_MinVersion(t *testing.T) {
+	c := &ClientConfig{
```

Comment:
Nothing crucial, just a suggestion: How about rewriting this as a data driven test (with `t.Run` sub-tests), that tests all valid TLS version strings and maybe an invalid version string? That would be more robust :)

Reply:
The same suggestion should hold for `TestGetTLSConfig_CipherSuites` for what it's worth.